pandas/tests/io/pytables/test_timezones.py

```diff
@@ -296,6 +296,16 @@ def test_timezones_fixed(setup_path):
         tm.assert_frame_equal(result, df)
 
 
+@pytest.mark.parametrize("dtype", ["datetime64[ns, UTC]", "datetime64[ns, US/Eastern]"])
```

Comment:
next PR can you change this to use the tz_aware_fixture

Reply:
Done